.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+node_modules
```

Comment:
Why this double? Also, if you've had to delete the .env file, add it in .gitignore.